vue/components/ui/molecules/search-persistent/save-filter-modal.vue

```diff
@@ -95,25 +121,148 @@ export const SaveFilterModal = {
             return (
                 this.searchData &&
                 this.filterNameData &&
-                Object.keys(this.tenacyValuesData).length > 0
+                Object.keys(this.tenancyCheckboxValuesData).length > 0 &&
+                this.isTenancyFormValid
             );
         },
         filters() {
-            return Object.keys(this.tenacyValuesData).map(tenancy => ({
+            return Object.keys(this.tenancyCheckboxValuesData).map(tenancy => ({
                 name: this.filterNameData,
                 value: this.searchData,
-                tenancy: tenancy,
+                tenancy: this.getSelectedTenancy(tenancy),
                 context: "" // TODO
             }));
+        },
+        isTenancyEmpty() {
+            return this.tenancyItemsData.length === 1;
+        },

```

Comment:
Inline this function. I understand what you mean but saying "empty" in the method and then having `=== 1` is a bit odd, but if you have it in the proper context (HTML) then it becomes clear what you're doing.